### ISSUE TITLE
[Taxon]: Measure_Length_Form_Angle

### ISSUE THREAD

**Issue by BamaBlacksmith** [NCSLI-MII/measurand-taxonomy#40]
### Contact Details

michael.linn@indysoft.com

### What is the process type?

Measure

### Measurand

Measurement of a dimensional angle. calibration of Angle Blocks, Angle Gages, Sine Bar, Sine Plates, inclinometers, levels, etc

### Kind of quantity

geometric dimensional Angle (as opposed to electrical phase angle) 

### Does the measurand require a special token?

None

### Taxon Name

Measure.Length.Form.Angle

**Comment by SchwartzMichael** [comment]
I thought this one was already created.  But I don't see it in the repository. 

I think it would look something like
Source.Angle or Source.PlaneAngle

I think the angle is measured on a 2-dimensional plane.   
Degrees would be the primary unit, with conversion to radian, gradian, angular mils

The big question is how different an Angle Block, Angle Gauge, Sine Bar, Sine Plate measurements are.  
Would these be a different Type Parameter meaning they are fundamentally the same.

**Comment by ryanmwhitephd** [comment]
I agree @SchwartzMichael These are fundamentally the same -- the sum of angles on a rotational plane sums to 360 degrees. Parameters may be used to indicate the Type, polygon, sine bar, etc... 

I'd probably start with just Source.Angle.
